website/docs/cloud-docs/integrations/aws-service-catalog/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+page_title: Terraform Cloud for AWS Service Catalog
+description: >-
+  Provision infrastructure with HashiCorp Terraform Cloud inside AWS Service Catalog
+---
+
+# Terraform Cloud for AWS Service Catalog overview
```

Comment:
Same question / suggestion as above: Should we mirror the nav's title here and generalize this a bit? Also, (nitpick) I believe that the "o" in "overview" should be capitalized here for consistency:
```suggestion
# AWS Service Catalog Integration Overview
```

Reply:
I did this because I was actually following this page as an example: https://developer.hashicorp.com/terraform/cloud-docs/integrations/kubernetes